.github/workflows/integration-tests.yaml

```diff
@@ -1,32 +1,23 @@
-name: Trigger Integration Tests
+name: Integration Tests (Pull Request)
```

Comment:
Why is this named `(Pull Request)` but only runs on `master` and `it/` branches?